queries.yaml

```diff
@@ -166,7 +177,6 @@ layers:
     area-inclusion-threshold: 1
   transit:
     template: transit.jinja2
-    start_zoom: 5
```

Comment:
The smallest feature min_zoom is already 5 so relying on feature min_zoom alone (not guarding here in the layer config) should be fine.